website/content/en/status/report-2023-07-2023-09/freebsd-foundation.adoc

```diff
@@ -0,0 +1,156 @@
+* Continued planning the link:https://freebsdfoundation.org/news-and-events/event-calendar/november-2023-freebsd-vendor-summit/[November 2023 FreeBSD Vendor Summit], taking place November 2-3, 2023, in San Jose, CA
+* Continued to administer our Google Summer of Code program.
+* Published link:https://freebsdfoundation.org/news-and-events/newsletter/freebsd-foundation-update-july-2023/[July Newsletter]
+* Additional Blog Posts
+** link:https://freebsdfoundation.org/blog/advocating-at-events-may-2023-freebsd-dev-summit-and-bsdcan/[Advocating at Events: May 2023 FreeBSD Dev Summit and BSDCan]
+** link:https://freebsdfoundation.org/blog/top-ten-reasons-to-upgrade-to-freebsd-13-2/[Top Ten Reasons to Upgrade to FreeBSD 13.2]
+** link:https://freebsdfoundation.org/blog/july-2023-software-development-projects-update/[July 2023 Software Development Projects Update]
+** https://freebsdfoundation.org/blog/freebsd-for-research-cheri-morello/[FreeBSD for Research: CHERI/Morello]
+** Meet the FreeBSD Google Summer of Code Students
+*** link:https://freebsdfoundation.org/blog/meet-the-2023-freebsd-google-summer-of-code-students-soobin-rho/[Soobin Rho]
+*** link:https://freebsdfoundation.org/blog/meet-the-2023-freebsd-google-summer-of-code-students-raghav-sharma/[Raghav Sharma]
+*** link:https://freebsdfoundation.org/blog/meet-the-2023-freebsd-google-summer-of-code-students-sudhanshu-mohan-kashyap/[Sudhanshu Mohan Kashyap]
+*** link:https://freebsdfoundation.org/blog/meet-the-2023-freebsd-google-summer-of-code-students-aymeric-wibo/[Aymeric Wibo]
+** link:https://freebsdfoundation.org/blog/meet-the-summer-2023-university-of-waterloo-co-op-student-naman-sood/[Meet The Summer 2023 University of Waterloo Co-Op Student: Naman Sood]
+** link:https://freebsdfoundation.org/blog/meet-freebsd-foundation-2023-summer-intern-jake-freeland/[Meet FreeBSD Foundation 2023 Summer Intern: Jake Freeland]
+* FreeBSD in the News:
+** link:https://freebsdfoundation.org/news-and-events/latest-news/freebsd-foundation-joins-osis-open-policy-alliance/[FreeBSD Foundation Joins OSI's Open Policy Alliance]
+** link:https://freebsdfoundation.org/news-and-events/latest-news/hackernoon-5-reasons-we-use-open-source-freebsd-as-our-enterprise-os/[Hackernoon: 5 Reasons We Use Open Source FreeBSD as Our Enterprise OS]
+** link:https://freebsdfoundation.org/news-and-events/latest-news/what-the-dev-podcast-the-evolution-of-the-freebsd-project/[What the Dev Podcast: The Evolution of the FreeBSD Project]
+
+We help educate the world about FreeBSD by publishing the professionally produced FreeBSD Journal.
+As we mentioned previously, the FreeBSD Journal is now a free publication.
+Find out more and access the latest issues at link:https://www.freebsdfoundation.org/journal/[].
+
+You can find out more about events we attended and upcoming events at link:https://www.FreeBSDfoundation.org/news-and-events/[].
```

Comment:
```suggestion
You can find out more about events we attended, and upcoming events, at link:https://www.FreeBSDfoundation.org/news-and-events/[].
```

Reply:
I don't think either of these commas is standard.  The first is before a conjunction that introduces a dependent clause.  Grammarly agrees.

Reply:
Commas removed, thanks. 

URL updated (one of the redirects), sorry, I might have missed this one a few hours ago.